src/category_theory/opposites.lean

```diff
@@ -271,6 +271,26 @@ protected def op (α : X ≅ Y) : op Y ≅ op X :=
   hom_inv_id' := quiver.hom.unop_inj α.inv_hom_id,
   inv_hom_id' := quiver.hom.unop_inj α.hom_inv_id }
 
+/-- The isomorphism obtained from an isomorphism in the opposite category. -/
+@[simps] def iso.unop {X Y : Cᵒᵖ} (f : X ≅ Y) : (Y.unop ≅ X.unop) :=
```

Comment:
This and the below are already in the `iso` namespace!
```suggestion
@[simps] def unop {X Y : Cᵒᵖ} (f : X ≅ Y) : (Y.unop ≅ X.unop) :=
```

Reply:
Thanks